frontend/src/scenes/funnels/FunnelViz.tsx

```diff
@@ -64,6 +68,13 @@ export function FunnelViz({
         }
     }
 
+    function buildChart(): void {
+        // Build chart only for the selected display view
+        if (!featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] || filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+            return buildFlowChart()
+        }
```

Comment:
Well, I'd still merge these two "build*" functions... and is this `filters.display === ACTIONS_LINE_GRAPH_LINEAR` relevant still?

Reply:
Merged & added a brief comment to clarify.